system.go

```diff
@@ -0,0 +1,23 @@
+package utils
+
+import (
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+// StopEtcd stops etcd on a specific host
+func StopEtcd(node TestbedNode) error {
+	log.Infof("Stopping etcd")
+	return node.RunCommand("pkill etcd && rm -rf /tmp/etcd")
+}
+
+// StartEtcd starts etcd on a specific host.
+func StartEtcd(node TestbedNode) error {
+	log.Infof("Starting etcd")
+
+	_, err := node.RunCommandBackground("nohup etcd -data-dir /tmp/etcd </dev/null &>/dev/null &")
```

Comment:
this is done, PTAL @mapuri @jainvipin, I think this is ready for merge (please merge it for me if you are both comfortable)